### PR TITLE
WiFi workshop

### PR DESCRIPTION
**Pull request by lulf**
@ctron FYI, I'm stealing your previous work like crazy. I'd like to maybe take a different angle on this workshop though, and maybe you could help out with this @jbtrystram: I think it would be cool if the business application in this case could be a browser app that talked to drogue cloud directly using the websockets integration. I'm not sure how viable that is with auth etc.

On the device side, I'd need to add support for the temperature reading using the microbit on-board sensor, and change the examples to report that value instead of some dummy value.

On the 'making changes' to the device side, I was thinking of introducing an actor that parsed responses and extracting any commands from it, and applying it (printing 'ON'/'OFF' on the display for instance).

### CODE REVIEW

File: modules/esp8266-wifi/pages/index.adoc [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+== Pre-requisites
+
+This workshop requires additional hardware, as we need an embedded device with a WiFi modem.
+
+Additionally to the common pre-requisites, you will need:
+
+* A Drogue Device supported development board and WiFi modem
+* A WiFi network
+* A publicly accessible Drogue IoT Cloud instance running version 0.5+
```
Comment (ctron) [inline modules/esp8266-wifi/pages/index.adoc:17]: Maybe point to the sandbox if people want to focus on the embedded side.

Comment (lulf) [reply modules/esp8266-wifi/pages/index.adoc:17]: Agreed, we've got the drogue cloud setup covered already elsewhere I guess.

**Comment by jbtrystram** [comment]
A web app getting events with websockets is very doable :) I'll try to mock something soon.
If we're doing temperature data let's do something a rolling  avg and determine if the trend is warming or cooling

**Comment by lulf** [comment]
I've written up the bits for device side + the wasm application consuming events. @dejanb When you have some content, just put it in the db-and-grafana.adoc file or wherever you feel is more appropriate.

File: modules/wifi-websockets/pages/db-and-grafana.adoc [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+Let's deploy this example and configure it to store and visualize our device temperature data. For that we will need Kubernetes cluster and https://helm.sh/[Helm] available on our machine. In this case we will use https://minikube.sigs.k8s.io/[Minikube], but any other will work just fine.
+
+First, you need to start the Minikube
+
+[source]
+----
+minikube start --addons ingress
+----
+
+Considering, we have our Drogue cloud properly configured in the previous section, we need to create an _Access Token_ to connect to the sandbox.
```
Comment (lulf) [inline modules/wifi-websockets/pages/db-and-grafana.adoc:16]: cloud -> Cloud?

File: modules/wifi-websockets/pages/db-and-grafana.adoc [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+[source]
+--
+drg admin tokens create
+
+A new Access Token was created:
+
+drg_TOKEN_XXX
```
Comment (lulf) [inline modules/wifi-websockets/pages/db-and-grafana.adoc:24]: Maybe drg_HASH_xxx . Using Token might be confusing it with the whole which is a token as well.

File: modules/wifi-websockets/pages/db-and-grafana.adoc [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+== Future
+
+The current Helm chart is just a first step in providing easy getting started telemetry application for Drogue cloud. While it works nicely in the scenario we just described, there are a lot more we can improve upon:
+
+* Provide more integration options - current implementation supports only web sockets. We'd like to provide more connectivity options (MQTT and Kafka) and more resilience to connection handling
+* Run in serverless environment - Many cloud environments, like https://developers.redhat.com/developer-sandbox[Openshift developer sandbox] provide Servlerless/Knative support for its users. We should leverage it when we can
+* Run outside of Kubernetes - Although Kubernetes is very popular today, there are many situation where you want to run an application like this without the access to the cluster. We should provide a way to run the application in plain Docker/Podman installations.
+
+So stay tuned!
```
Comment (lulf) [inline modules/wifi-websockets/pages/db-and-grafana.adoc:65]: Should this be moved to the summary section perhaps?

**Review by lulf — COMMENTED** [review]
@dejanb I added some comments

**Pull request merged**